examples/testTag/testTagOnClassIgnoredInTestClass.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace TestTagOnConstructor;
+
+use DaveLiddament\PhpLanguageExtensions\TestTag;
+
+#[TestTag]
+class Person
+{
+    public function __construct()
+    {
+    }
+
+    public static function create(): Person // No Error, class can interact with itself
+    {
+        return new Person(); // No Error, class can interact with itself
+    }
+}
+
+class PersonTest
+{
+    public function newInstance(): Person
+    {
+        return new Person(); // ERROR
```

Comment:
In this example I think it is OK to call `new Person` as I assume `PersonTest` is deemed test code?